### PR TITLE
New package: LGUG2Z.whkd version 0.1.1

### PR DESCRIPTION
**Pull request by LGUG2Z**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96411)

### CODE REVIEW

**Comment by Trenly** [comment]
```
2023-02-12 07:34:13.237 [FAIL] Installer failed security check. Url: https://github.com/LGUG2Z/whkd/releases/download/v0.1.0/whkd-0.1.0-x86_64.msi Result: 0x80004005
```

**Comment by LGUG2Z** [comment]
@Trenly Thanks for sharing the additional context- I've tried Googling this error but did not find anything that seemed like it is appropriate in this context.

I turned on Windows AV locally and saw this issue when trying to install again:

image

This seems to be quite a clear case of a false positive, most likely due to the use of UDS sockets: https://github.com/ninja-build/ninja/issues/2222#issuecomment-1400663164

Many more examples here: https://github.com/search?q=+trojan%3Awin32%2Fwacatac.h%21ml&type=issues

Please let me know how we can proceed!

**Comment by stephengillie** [comment]
The program installed normally in a VM, but errored when launched:
image

Is a dependency missing?

**Comment by stephengillie** [comment]
> Ah I see! It is looking for a configuration file (whkdrc) that does not exist in the VM.
> 
> Typically this is a valid failure case (the hotkey daemon cannot run without a user-provided configuration file file), but I'm open to updating this to return a non-error exit code if that is what is required to pass the pipeline validation checks here.

Nice find. How does a user normally obtain this configuration file? Is it generated by this program, by a dependency, or by a separate program like a configuration file writer?

**Comment by LGUG2Z** [comment]
It is written by hand by the user; there isn't really any "default" configuration in this use case because we don't want to surprise the user by enabling systemwide hotkeys in a sample configuration that they don't fully understand or that may not make sense on their system

**Comment by LGUG2Z** [comment]
I have released a new version which exits without an error code and leaves a message informing the user of the need to create a configuration file in the `~/.config` directory. I believe that this should enable the program to run cleanly in the VM on the validation pipelines. 🤞

**Comment by sitiom** [comment]
> I have released a new version which exits without an error code and leaves a message informing the user of the need to create a configuration file in the `~/.config` directory. I believe that this should enable the program to run cleanly in the VM on the validation pipelines. 🤞

@LGUG2Z You can close this PR and make a new PR of the new version.

**Comment by LGUG2Z** [comment]
Last pipeline failure looks like this issue again... https://github.com/microsoft/winget-pkgs/pull/96411#issuecomment-1427235555

File: manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LGUG2Z.whkd
+PackageVersion: 0.1.1
+PackageLocale: en-US
+Publisher: LGUG2Z
```
Comment (ItzLevvie) [inline manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.locale.en-US.yaml:7]:
```suggestion
Publisher: Jade Iqbal
```
This must match Control Panel.

File: manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- whkd
+ReleaseDate: 2023-02-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/LGUG2Z/whkd/releases/download/v0.1.1/whkd-0.1.1-x86_64.msi
+  InstallerSha256: 1dc9798953d1b3eff35db4fba73e1bb074e7ffb8e0e9b224206c340d78945b51
+  ProductCode: '{EB940FAD-0B97-47AE-A3ED-1F2F5972B8D8}'
```
Comment (ItzLevvie) [inline manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.installer.yaml:18]:
```suggestion
  ProductCode: '{F0EDC6C1-CA34-4BC7-B34A-8476D625DC4E}'
```
This must match Windows Registry.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/96411#pullrequestreview-1300616116

**Comment by LGUG2Z** [comment]
Again I can't get much information from what is publicly shown in the Azure pipelines, but both the local validation and installation of this manifest are working for me without issue

**Comment by LGUG2Z** [comment]
https://www.virustotal.com/gui/file/1dc9798953d1b3eff35db4fba73e1bb074e7ffb8e0e9b224206c340d78945b51/detection

Above is a report from VirusTotal showing that Windows is the only security vendor falsely labeling this file as containing a virus.

I have also submitted a false positive report to Microsoft: 
image

**Comment by LGUG2Z** [comment]
image

The false positive should be fixed now, can we try the installation validations again?

**Comment by sitiom** [comment]
Try closing and reopening the PR to retrigger it.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**